stock_measuring_device/static/src/js/measuring_wizard.esm.js

```diff
@@ -0,0 +1,32 @@
+import {registry} from "@web/core/registry";
+
+export const measuringWizardNotificationService = {
+    dependencies: ["action", "bus_service", "orm"],
+    start(env, {action, bus_service, orm}) {
+        this.action = action;
+        this.bus_service = bus_service;
+        this.orm = orm;
+        const self = this;
+        this.bus_service.subscribe("notify_measuring_wizard_screen", (notification) => {
+            self.notifyMeasuringWizardScreen(notification).then((result) => {
```

Comment:
```suggestion
        this.bus_service.subscribe("notify_measuring_wizard_screen", (notification) => {
            this.notifyMeasuringWizardScreen(notification).then((result) => {
```

In modern js there's no more need to do `const self = this`.
Arrow functions do not have their own `this` binding. They inherit `this` from the surrounding scope

Reply:
While testing, I got that `this` was `undefined` and `this.notifyMeasuringWizardScreen()` crashed